DataFormats/HcalDigi/interface/HcalUMNioDigi.h

```diff
@@ -0,0 +1,47 @@
+#ifndef DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H
+#define DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H 1
+
+#include <vector>
+#include <stdint.h>
```

Comment:
Use <cstdint> mainly because you are coding in C++, not C. It will be available in the global namespace, look at the implementation. IIRC, typically C++ version of header will include C header and then also define typedefs in `std::` namespace.
```
 40 #if _GLIBCXX_HAVE_STDINT_H
 41 # include <stdint.h>
 42 #endif
 43
 44 #ifdef _GLIBCXX_USE_C99_STDINT_TR1
 45
 46 namespace std
 47 {
 48   using ::int8_t;
 49   using ::int16_t;
 50   using ::int32_t;
 51   using ::int64_t;
[..]
```